frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -259,29 +262,27 @@ const FileHandler = ({
             if (response?.id) {
                 setReportId(response.id);
                 setSuccessTimestamp(response.timestamp);
-                event.currentTarget.reset();
+                event?.currentTarget?.reset && event.currentTarget.reset();
```

Comment:
this was throwing a silent error when the currentTarget did not have a reset method